### DISCUSSION TITLE
Default font family

### DISCUSSION THREAD

**Discussion by Phunkhouse** [standardnotes/forum#2532] [Suggestions]
Hi, is it possible to set default font family for whole app? I am using Plus editor and this thing kind of grabbing my time. Also I am a user of macOS, Windows and iOS, so every system have different native font. Thanks

**Comment by atmoio** [comment]
This isn't possible currently. It's really only the Plus editor that supports custom fonts. As for the whole app, do you mean an interface font, or just for the editor?

**Comment by pwseo** [comment]
@moughxyz, I think OP was referring to the default UI font.

Still, wouldn't it be possible to change the default font for the plain-text editor?
I'm fond of monospaced fonts for plain-text notes, but the current font stack specified by the default style could be better (for me, of course): it defaults to Liberation Mono for me, when there are other, more wonderful fonts to choose from (like Source Code Pro, IBM Plex Mono).
Instead of having to create an extension to override such a CSS variable, one should be able to specify it directly in the app's settings (ideally, this should not be tied to one's account information, but rather it should be a local variable). Perhaps this could be an "advanced setting".

Please let me know what you think about this.
